### ISSUE TITLE
Add aggregate functions to query builder

### ISSUE THREAD

**Issue by asimpson** [grafana/sentry-datasource#73]
In order to support additional alerting functionality (e.g. fire an alert when the number of issues crosses an interval) we should support aggregate [functionality](https://grafana.com/docs/grafana/latest/fundamentals/timeseries/#aggregating-time-series) in the query builder.

**Comment by vanchaxy** [comment]
Isn't this already covered by [reduce expression](https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/expression-queries/#reduce)?

E.g: this alert will fire when you have more than 300 issues in 3 hours
<img width="1099" alt="image" src="https://github.com/grafana/sentry-datasource/assets/36510601/6c612295-4818-40a8-909b-8d23df601ce9">

**Comment by asimpson** [comment]
@vanchaxy You are correct that reduce is the right expression but if you try a Issue query it won't work.

**Comment by kesik87** [comment]
Having same issue with creating grafana alert. Creating one for "stats" query type doesn't work for me as there is no filtering by environment.
